src/assets.json

```diff
@@ -282,5 +282,27 @@
         "expired": false
       }
     ]
+  },
+  "137": {
+    "2xdpi": [
+      {
+        "name": "October26",
```

Comment:
we need to keep the test assets in separate file, just switch that file from the source on sdk init for uses

Reply:
For testing the actual functionality of the functions i implemented I merged it into one file.
Later when we deploy I will remove it.